src/client/components/sidebar/sidebar.js

```diff
@@ -42,13 +49,13 @@ class Sidebar extends React.Component {
           </Button>
           <hr />
           <Leaderboard playerID={this.props.playerID} scores={this.props.G.scores} names={this.props.names} cards={getDealtCardsForPlayers(this.props.G.order, this.props.G.dealt)} />
-          <hr />
-          <Button color="secondary" size="lg" block disabled={
+          <p className={(isLastToPass) ? "visible" : "invisible"} >You are the last one to pass!</p>
```

Comment:
As written in the other PR, it's better to use conditional rendering than making it invisible via CSS.

Reply:
Could you also move this line a little bit down? It looks to be a bit to high.
![Screenshot from 2021-07-15 22-24-35](https://user-images.githubusercontent.com/10673021/125852960-179e48ef-f937-48ab-b6dc-56e11437d6e7.png)
And maybe use a color that is better readable on a white background?